app/components/view-question-data-text/macro.njk

```diff
@@ -1,11 +1,7 @@
-{% from 'components/label/macro.njk' import label %}
-
 {% macro viewQuestionDataText(params) %}
   {% if params.questionData %}
-    <div data-test-id="view-question-data-text-{{params.questionId}}">
-      {{ label({
-        "text": params.questionData
-      }) }}
+    <div data-test-id="view-question-data-text-{{ params.questionId }}" class="bc-c-data-text {{ params.classes }}">
+      {{ params.questionData }}
```

Comment:
do we not need to put it around <p> tags?

Reply:
erm what?

Reply:
😂 it didn't show the `<p>` tag

Reply:
nah it is surrounded in a div